oak_crypto/proto/v1/BUILD

```diff
@@ -39,11 +39,17 @@ java_proto_library(
     deps = [":crypto_proto"],
 )
 
+java_lite_proto_library(
```

Comment:
@k-naliuka or @ipetr0v, any pointers on this failure from xtask run-bazel-tests?

```
ERROR: /workspace/oak_crypto/proto/v1/BUILD:27:14: in @_builtins//:common/java/proto/java_lite_proto_library.bzl%java_lite_proto_aspect aspect on proto_library rule //oak_crypto/proto/v1:crypto_proto

Error: @_builtins//:common/java/proto/java_lite_proto_library.bzl%java_lite_proto_aspect aspect on proto_library has to declare 'proto' as a required fragment in target configuration in order to access it. Please update the 'fragments' argument of the rule definition (for example: fragments = ["proto"])
```

Reply:
mitigated by removing the android_library